.gitleaksignore

```diff
@@ -0,0 +1,106 @@
+77d07108ea198b6b96eaeb71e296ffd0c3e6e3fa:tests/test_packages/test_connections/test_p2p_libp2p_client/test_errors.py:generic-api-key:233
```

Comment:
Did you review all for non-critical issues @dvilelaf ?

Reply:
Yes: most entries come from tests, docs and examples. Only these are from other files :
```
tac_controller/vendor/fetchai/connections/soef/connection.yaml
tac_participant_1/vendor/fetchai/connections/soef/connection.yaml
tac_participant_2/vendor/fetchai/connections/soef/connection.yaml
packages/fetchai/agents/tac_controller_contract/aea-config.yaml
packages/fetchai/connections/soef/connection.yaml
packages/fetchai/connections/soef/connection.py
tac_controller/default_private_key.pem
tac_participant_one/default_private_key.pem
my_new_agent/default_private_key.pem
m_agent/default_private_key.pem
```